js/app.js

```diff
@@ -1,16 +1,28 @@
 // Enemies our player must avoid
-var Enemy = function() {
-    // Variables applied to each of our instances go here,
-    // we've provided one for you to get started
+// in this file we have to create a render methods for player and enemies
 
+var Enemy = class {
+    constructor(speed , sprite , x , y , d){
+        this.speed = speed;
+        this.sprite = sprite;
+        this.x = x;
+        this.y = y;
+        this.d = d;
+    }
     // The image/sprite for our enemies, this uses
     // a helper we've provided to easily load images
-    this.sprite = 'images/enemy-bug.png';
 };
 
 // Update the enemy's position, required method for game
 // Parameter: dt, a time delta between ticks
 Enemy.prototype.update = function(dt) {
+
+    this.x = this.x + this.speed * this.d;
+    this.x = this.x%500;
```

Comment:
- [ ] magic numbers 500? It can be defined at top as something like `CANVAS_WIDTH`.